packages/breadcrumbs/src/lib/Breadcrumbs.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { Breadcrumbs, BreadcrumbsProps as MuiBreadcrumbsProps, Typography } from '@mui/material';
+import {NavigateNext} from '@mui/icons-material'
```

Comment:
the mui docs recommend doing a more specific import for bundle sizes.
`import NavigateNext from '@mui/icons-material/NavigateNext';`